modules/javafx.controls/src/test/java/test/javafx/scene/chart/XYChartTest.java

```diff
@@ -82,25 +82,25 @@ public void testTickMarksToString() {
         startApp();
         Font f = yaxis.getTickLabelFont();
         // default caspian value for font size = 10
-        assertEquals(10, new Double(f.getSize()).intValue());
-        assertEquals(10, new Double(AxisShim.get_measure(yaxis).getFont().getSize()).intValue());
+        assertEquals(10, Double.valueOf(f.getSize()).intValue());
```

Comment:
Any reason to convert `f.getSize()` to `int` and then to `Double`? Is it for flooring the `double`?

Same for the other places.

Reply:
`f.getSize()` returns a double value, the statement can also be written as `assertEquals(10, f.getSize(), 0f);`
But, as this change is a cleanup and is intended to remove the calls to deprecated constructors, I think it should be limited to changing calls to autobox or to use `valueOf()` method.